net/nginx-util/src/px5g-openssl.hpp

```diff
@@ -109,7 +109,7 @@ auto gen_eckey(const int curve) -> EVP_PKEY_ptr
 
     EC_GROUP_set_point_conversion_form(group, POINT_CONVERSION_UNCOMPRESSED);
 
-    auto eckey = EC_KEY_new();
+    auto *eckey = EC_KEY_new();
```

Comment:
Why?

Reply:
Just for style (clang-tidy: readability-qualified-auto and llvm-qualified-auto suggest it)

Reply:
OK. Me myself I don't like that check.

```
auto a = 5;
auto b = "test";
a = 6;
b = "hello";

const auto c = 5;
const auto d = "j";
a = 6; //error
b = 6; /error

const auto e = 5;
const auto *f = "hmm";
e = 6; //error
f = "uhhh"; // works......
```

Reply:
I see, I will think about it (`auto *const` would be an option, too) Thank you :-)

Reply:
So, I decided to use `auto` again (not only here).